### PR TITLE
4991 add loading messages for reports

### LINKED ISSUE #4991: Reports initially render very slow and without loading indicators
## What went wrong? 😲

First time rendering reports looks like this:
https://github.com/user-attachments/assets/77c81a99-413d-4d1c-9fa0-8e63f4f3da9a

Subsequent reports are faster... but i still think loading indication would help!

## Expected behaviour 🤔

At least let me know the report is coming, rather than `not found` 😢 

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. On APK, generate a report
2. Watch and wait 😁 

## Your environment 🌱

- Open mSupply Version: v2.3.0 RC5
- Legacy mSupply Central Server Version:

- Platform: APK

- Database type:

### PR DESCRIPTION
**Pull request by fergie-nz**
Fixes #4991 

# 👩🏻‍💻 What does this PR do?

Adds loading checks for reports

## 💌 Any notes for the reviewer?

# 🧪 Testing

Go to reports
Using dev tools, increase throttling
See loading messages that make sense rather than a blank page.

# 📃 Documentation

- [x] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour

### CODE REVIEW

**Comment by fergie-nz** [comment]
Wondering if I should make a similar PR into our RC? Or leave this for next release?

**Comment by jmbrunskill** [comment]
> Wondering if I should make a similar PR into our RC? Or leave this for next release?

Yes, the milestone is for the RC so we should create the PR into that branch instead.

Check you console logs too :)

**Review by jmbrunskill — CHANGES_REQUESTED** [review]
Please re-raise this for v2.3.0-RC

File: client/packages/common/src/intl/locales/en/common.json [hunk 1]
```diff
@@ -1597,5 +1597,6 @@
   "warning.cannot-create-placeholder-units": "There is a total of {{allocatedQuantity}} units available. Unable to allocate all {{requestedQuantity}} units.",
   "warning.caps-lock": "Warning: Caps lock is on",
   "warning.field-not-parsed": "{{field}} not parsed",
-  "warning.nothing-to-supply": "Nothing left to supply!"
+  "warning.nothing-to-supply": "Nothing left to supply!",
+  "messages.loading-report": "Attempting to load report..."
```
Comment (jmbrunskill) [inline client/packages/common/src/intl/locales/en/common.json:1601]:
```suggestion
  "messages.loading-report": "Loading report..."
```
Mabye a simplier wording?

**Comment by fergie-nz** [comment]
> Please re-raise this for v2.3.0-RC

https://github.com/msupply-foundation/open-msupply/pull/5038

**Comment by jmbrunskill** [comment]
Duplicate

**Pull request closed without merging**